docs/devsite-help/support.md

```diff
@@ -0,0 +1,223 @@
+# Support
+
+The vast majority of the code in the Google Cloud Libraries is autogenerated. If
```

Comment:
```suggestion
The vast majority of the code in the Google Cloud Libraries repository is autogenerated. If
```

Reply:
Do we need this one? Let's talk about it.

Reply:
You mean the whole sentence? Yep, I would be fine with removing it, there's a whole section about Server side issues.

Reply:
Nope, I specifically meant "repository". I don't think we need to distinguish there.